rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -229,9 +234,9 @@ void game_list_frame::FixNarrowColumns()
 
 void game_list_frame::ResizeColumnsToContents(int spacing)
 {
-	if (!m_gameList)
-	{
-		return;
+	if (!m_gameList)
+	{
+		return;
```

Comment:
Any idea what's up with these diffs?

Reply:
Yeah... Visual Studio messed up line endings. I tried to do something about that, but in that file there's a mix of LF and CRLF. If this is a problem i'll try to rewrite my changes starting from the original file.

Reply:
That's interesting.
I was under the impression that git, by default on Windows, would convert them to LF on checkin, and CRLF on checkout.
Unless there's some sort of override in `.gitattributes`, or `core.autocrlf` was messed with.

Reply:
I had similar issues when editing the file too, I fixed it by submitting the line changes as another commit and dropping it out from the PR. I whonder whether we should just include the line fixes as having to work around them every time the file is modified seems very cubersome. Maybe @Megamouse can explain what went wrong with the line endings, according to git blame he's the guy who added those sections there.

Reply:
there's nothing wrong with my VS settings. it's the git settings. I save files in the same format in which they are provided.
I do that so I dont have unnecessary changes like this. idk who changed the settings but there's no reason to start formatting random sections all of a sudden

Reply:
As someone told me on discord, you can use blame -w. I know it's not the solution but everytime i commit my changes the git do what he wishes, so i think is better to change that file now and don't care about it again next time.

Reply:
well alright. but could you please put it first so it's easier to to do a mixed reset on that commit?